packages/fx-core/src/plugins/resource/appstudio/constants.ts

```diff
@@ -119,8 +119,8 @@ export const TEAMS_APP_MANIFEST_TEMPLATE_FOR_MULTI_ENV = `{
   ],
   "validDomains": [],
   "webApplicationInfo": {
-      "id": "{appClientId}",
-      "resource": "{webApplicationInfoResource}"
+      "id": "{{profile.fx-resource-aad-app-for-teams.clientId}}",
+      "resource": "{{{profile.fx-resource-aad-app-for-teams.applicationIdUris}}}"
```

Comment:
I see you're using triple `'{'` and `'}'` in some placeholders, while mustache uses double `'{'` and `'}'`for the tag to be resolved.